pyproject.toml

```diff
@@ -27,7 +27,7 @@ httpx = "^0.24.0"
 [tool.poetry.dev-dependencies]
 pytest = "^7.2"
 pytest-asyncio = "^0.20"
-pytest-mock = "^3.10"
+pytest-mock = "^3.14.0"
```

Comment:
we need to run `poetry lock` after this change

Reply:
Ah, thank you. Will do.